public/scripts/slash-commands/SlashCommand.js

```diff
@@ -36,6 +36,7 @@ export class SlashCommand {
      * @param {string} [props.helpString]
      * @param {boolean} [props.splitUnnamedArgument]
      * @param {Number} [props.splitUnnamedArgumentCount]
+     * @param {boolean} [props.rawQuotes]
```

Comment:
I know Lenny didn't do it, but can we start introducing jsdocs for what the properties actually mean?

For example, that this one only applies to the unnamed arg might be a useful information given in a jsdoc.